app/jobs/dttp/check_consistency_job.rb

```diff
@@ -22,11 +25,11 @@ def trainee
     end
 
     def contact_conflict
-      consistency_check.contact_last_updated_at != dttp_contact_updated_date
+      consistency_check.contact_last_updated_at.utc < dttp_contact_updated_date&.to_datetime&.utc
```

Comment:
I think `Dttp::Contact#updated_at` should be returning a datetime object instead of having to do it here.

Reply:
Yeah. I agree. I'll sort that out in another PR.